libs/components/lookup/testing/src/public-api.ts

```diff
@@ -12,8 +12,9 @@ export { SkyCountryFieldSearchResultHarnessFilters } from './modules/country-fie
 
 export { SkyLookupHarness } from './modules/lookup/lookup-harness';
 export { SkyLookupHarnessFilters } from './modules/lookup/lookup-harness-filters';
+export { SkyLookupSearchResultHarness } from './modules/lookup/lookup-search-result-harness';
+export { SkyLookupSearchResultHarnessFilters } from './modules/lookup/lookup-search-result-harness-filters';
 export { SkyLookupSelectionHarness } from './modules/lookup/lookup-selection-harness';
-export { SkyLookupSelectionHarnessFilters } from './modules/lookup/lookup-selection-harness-filters';
```

Comment:
This type is not used anywhere in our code base.

Reply:
Technically, yes. This is being applied to the `main` branch, and I didn't find a single instance of it in ADO.